Sources/CLI/Application.swift

```diff
@@ -304,8 +308,14 @@ struct Application: AsyncParsableCommand {
 extension Application {
     // Because we support plugins, we need to modify the help text to display
     // any if we found some.
-    static func printModifiedHelpText() {
-        let altered = Self.pluginLoader.alterCLIHelpText(
+    static func printModifiedHelpText(pluginLoader: PluginLoader?) async {
+        let original = Application.helpMessage(for: Application.self)
+        guard let pluginLoader else {
+            print(original)
+            print("PLUGINS: not available, run `container system start`")
+            return
+        }
+        let altered = pluginLoader.alterCLIHelpText(
```

Comment:
nit: could just re-use `original` from line 312.
```suggestion
        let altered = pluginLoader.alterCLIHelpText(original: original)
```